### PR TITLE
Desktop javascript

### PR DESCRIPTION
**Pull request by MohamedRoshdy2021**
in this Pr i have
- Removed the data from HTML into dynamic javascript objects 
- Added the layout for pop up window 
- Added the styling for the pop-up window

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @MohamedRoshdy2021 :wave: , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with no contribution from your partner as the project requirement asked that you do. Kindly work on this with your partner and submit a new PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Zuhaib042 — COMMENTED** [review]
Hi @MohamedRoshdy2021 @solog0039 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that is a pair-programming activity and while your partner has made a commit, but that commit relates only to his contact information. for pair-programming  activities, it is necessary that both partners commit actual code in the project and not just mere changings in the readme file. kindly consider getting actual code commit from your partner.
- [ ] Also there is merge conflict in your pull request, kindly consider solving this merge conflict before submitting again because Github doesn't check for linters if there is merge conflict in the pull request.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js.js [hunk 1]
```diff
@@ -26,3 +26,133 @@ list.forEach((e) => {
...
+    technologies2: ['Ruby', 'Bootstrap'],
+    liveLink: 'https://github.com/MohamedRoshdy2021',
+    sourceLink: 'https://example.com/source',
+    popuptext: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    popuptext2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industryLorem Ipsum has been the industrys <br></br> standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry.  ',
+  },
+  {
+    name: 'Multi-Post Stories',
+    image: './img2/Snapshoot Portfolio desk4.JPG',
```
Comment (franclobo) [inline js.js]: - [ ] Kindly correct the path of the image for a better UI. 👇🏽 

image

File: style.css [hunk 2]
```diff
@@ -87,6 +91,111 @@ p {
...
+  bottom: 0;
+  background-color: #fff;
+  padding: 0.7rem;
+  border-radius: 1rem;
+  border: 1px solid #dfe1e6;
+}
+
+.modal.active {
+  display: flex;
+}
```
Comment (franclobo) [inline style.css:112]: - [ ] The modal window looks incomplete in the mobile version, I recommend using `overflow-y: scroll` for a better UI 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MohamedRoshdy2021 and @solog0039, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Good UI ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please let your partner doing the required changes to warranty equal participation in the pair-programing project. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js.js (hunk 1 above)
Comment (MohamedRoshdy2021) [reply js.js]: but the path is correct and the images works just fine for me !!!  @franclobo

File: style.css (hunk 2 above)
Comment (Munsa1) [reply style.css:112]: - [ ] As requested by the previous reviewer, kindly fix the view of the modal popup in mobile view by adding an `overflow-y: scroll;` effect as it helps improve the user experience.

Your View with the CSS Property
image

Your view without the CSS Property
image

### Note that the style to the right on the first screenshot has the `overflow-y: auto` which makes the view more visible as the user is able to scroll down the modal page

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @MohamedRoshdy2021, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Passing Linters :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Kindly consider fixing your design to match the given design. For example the modal header color and the close icon in your design does not match the figma design

Figma Template
image

Your Design
image
## Also note the missing grey overlay in your design
<br/>

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -87,6 +91,113 @@ p {
...
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #fff;
+  padding: 0.7rem;
+  border-radius: 1rem;
+  border: 1px solid #dfe1e6;
+}
```
Comment (taldr27) [inline style.css:108]: - [ ] **[OPTIONAL]** Great job incorporating all the changes requested by the previous reviewer! I have one more suggestion **that is entirely optional** but **could enhance the overall appearance and align it closely with the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=dplvWZodUXxA9pZz-0)**. You might consider **adding a blurry background when the pop-up is open on both desktop and mobile devices**. This small addition can provide a more polished and visually appealing experience. 🤓

  image

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hello team @MohamedRoshdy2021, @solog0039, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README. 🤓
✅ Linters tests passing. 👍
✅ Incorporates feedback from previous code reviews. 💯
✅ Populating the popup dynamically. 🎯
✅ Popup functionality working fine. 🙌

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**